tools/engine_tool/lib/src/build_utils.dart

```diff
@@ -65,6 +65,31 @@ void debugCheckBuilds(List<Build> builds) {
   }
 }
 
+/// Transform the name of a build into the name presented and accepted by the
+/// CLI
+///
+/// If a name starts with '$OS/', it is a local development build, and the
```

Comment:
Can we expand this with 1 more sentence of examples of `$OS` (maybe mention it's `[Platorm.operatingSystem]`).

Reply:
Done.